### PR TITLE
Fix dead links

### PR DESCRIPTION
**Pull request by Samq64**
I found a dead link to the userscript docs, so I fixed it and any others I could find.

### CODE REVIEW

File: content/docs/reference/addon-api/addon.tab/addon.tab.traps.md [hunk 1]
```diff
@@ -12,8 +12,6 @@ aliases:
 
 `addon.tab.traps` is a public API for userscripts that can be used to obtain objects that are not otherwise available. The actual traps are coded at [prototype-handler](https://github.com/ScratchAddons/ScratchAddons/blob/master/content-scripts/prototype-handler.js).
 
-They share some APIs with "Once" traps, but is very different and is documented at [Fake redux state](Fake-redux-state) separately.
```
Comment (Samq64) [inline content/docs/reference/addon-api/addon.tab/addon.tab.traps.md:15]: @WorldLanguages Do you know what this was referring to?

Comment (mxmou) [reply content/docs/reference/addon-api/addon.tab/addon.tab.traps.md:15]: I checked the git history and apparently there was never a page with this name.

Comment (WorldLanguages) [reply content/docs/reference/addon-api/addon.tab/addon.tab.traps.md:15]: This line is good to remove then

**Review by WorldLanguages — APPROVED** [review]

File: content/docs/reference/addon-api/addon.tab/addon.tab.traps.md (hunk 1 above)
Comment (Hans5958) [reply content/docs/reference/addon-api/addon.tab/addon.tab.traps.md:15]: I look the old wiki and it seems that it is removed on v1.3.0. So, this is irrelevant and may be removed.

**Review by Hans5958 — APPROVED** [review]
LGTM

**Pull request merged**